src/components/heading/heading.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import Help from '../help';
 import Link from '../link';
-import Icon from '../icon';
 import tagComponent from '../../utils/helpers/tags';
 import './heading.scss';
+import StyledIcon from './heading.style';
```

Comment:
Can we export / import this as `StyledHeadingIcon`? Keeps things clear, as the Icon component itself is getting a re-write, so other places will be referring to StyledIcon as something different to this